examples/streamz/README.md

```diff
@@ -22,128 +22,155 @@ Run the following to build the image:
 docker build -f examples/streamz/Dockerfile -t clx-streamz:latest .
 `
 
-## Running cyBERT Inference Pipeline
+## Running CLX_Streamz Pipelines
 
-For simplicity, our inference pipeline is started when the Docker container is created using the above image. The following shows the format of the `docker run` to create your container.
+A docker container is created using the image above. The 'docker run' format to build your container is shown below. Note: volume binding to the container is an optional argument.
 
 **Preferred - Docker CE v19+ and nvidia-container-toolkit**
+
 ```
-docker run -it
---gpus '"device=0,1,2"' \
--p 8787:8787 \
---name cybert-streamz
--d cybert-streamz:latest \
---broker localhost:9092 \
---group_id streamz \
---input_topic input \
---output_topic output \
---model_file /path/to/model.pth \
---label_file /path/to/label.json \
---poll_interval 1s \
---max_batch_size 1000 \
---data /path/to/dataset
+docker run -it \
+    -p 8787:8787 \
```

Comment:
maybe bind to different host ports like 9787 and 9888 to avoid conflicts?

Reply:
@efajardo-nv done with changes